lookout/style/typos/corruption.py

```diff
@@ -64,36 +64,36 @@ def rand_swap(token: str) -> str:
     return token[:pos] + token[pos + 1] + token[pos] + token[pos + 2:]
 
 
-def _rand_typo(token_split: Tuple[str, str], typo_probability: float,
-               add_typo_probability: float) -> Tuple[str, str]:
+def _rand_typo(token_split: Tuple[str, str], typo_prob: float,
```

Comment:
Let's keep the longer names everywhere. Sometimes, `prob` is used as a shortened `probe` and it is confusing.